smartcar/smartcar.ino

```diff
@@ -0,0 +1,219 @@
+#include <MQTT.h>
```

Comment:
I don't think this file should be here either!

Reply:
File #include <vector> removed. 
Sketch does not compile without fiile #include <MQTT.h>

Reply:
The entire _file_ I commented on should be removed, it's a duplicate